test/AutoinstGeneral_test.rb

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env rspec
+
+require_relative "test_helper"
+
+Yast.import "AutoinstGeneral"
+Yast.import "Profile"
+
+describe Yast::AutoinstGeneral do
+  FIXTURES_PATH = File.join(File.dirname(__FILE__), 'fixtures')
+
+  describe "#ntp syncing in Write call" do
+    let(:profile_sync) { File.join(FIXTURES_PATH, 'profiles', 'general_with_time_sync.xml') }
+    let(:profile_no_sync) { File.join(FIXTURES_PATH, 'profiles', 'general_without_time_sync.xml') }
+
+    it "syncing hardware time if ntp server is set" do
+      Yast::Profile.ReadXML(profile_sync)
+      Yast::AutoinstGeneral.Import(Yast::Profile.current["general"])
+
+      expect(Yast::SCR).to receive(:Execute).with(
+        Yast::Path.new(".target.bash"),
```

Comment:
If this feature is *only for master* (SP1), you can use the shortcuts provided by [yast/rspec](http://www.rubydoc.info/github/yast/yast-ruby-bindings/master/Yast/RSpec/Shortcuts) and write `path` instead of `Yast::Path.new`.